zulipterminal/model.py

```diff
@@ -136,11 +136,15 @@ def __init__(self, controller: Any) -> None:
             # zulip_version and zulip_feature_level are always returned in
             # POST /register from Feature level 3.
             "zulip_version",
+            "alert_words",  # added alert_words
         ]
 
+        self.alert_words = self.fetch_alert_words()
```

Comment:
True. So I will change it to `self._alert_words` and add this function
```
  def get_alert_words(self):
        return self._alert_words
```
in `model.py`
and
`self.alerted_words = self. model.get_alert_words()`
in messages.py

Reply:
This would be the general idea, yes. Note that if the data is not immutable, or not copied, this still allows the caller to modify the alert words (even accidentally).